src/Raven.Studio/typescript/models/database/index/indexFieldOptions.ts

```diff
@@ -402,40 +434,86 @@ class indexFieldOptions {
         let placeHolder = null;
         const thisIndexing = this.indexing();
         const parentIndexing = this.parent() ? this.parent().indexing() : null;
+        const parentAnalyzer = this.parent() ? this.parent()?.analyzer() : null;
 
-        if (thisIndexing === "No" ||
-           (!thisIndexing && parentIndexing === "No")) {
+        const analyzerConfigurationKey = `Indexing.Analyzers.${thisIndexing === "Default" || thisIndexing === null ? `Default` : `${thisIndexing}.Default`}`;
+        
+        const databaseAnalyzerSetting = this.databaseIndexConfiguration?.[analyzerConfigurationKey];
+        
+        const localAnalyzerConfiguration: string | undefined = this.indexLocalConfiguration?.[analyzerConfigurationKey];
+
+        const hasDatabaseDefaultChanged =
+            databaseAnalyzerSetting?.effectiveValue() !== databaseAnalyzerSetting?.serverOrDefaultValue();
+        const isAnalyzerNameInDictionary = localAnalyzerConfiguration
+            ? this.analyzersNamesDictionary().some(
+                (analyzerItem) => analyzerItem.serverName === localAnalyzerConfiguration
+            )
+            : this.analyzersNamesDictionary().some(
+                (analyzerItem) => analyzerItem.serverName === databaseAnalyzerSetting?.effectiveValue()
+            );
```

Comment:
Also apply this aproach for getting `currentAnalyzerStudioName`

```suggestion
        const configuration = localAnalyzerConfiguration ?? databaseAnalyzerSetting?.effectiveValue();

        const isAnalyzerNameInDictionary = this.analyzersNamesDictionary().some((analyzerItem) => analyzerItem.serverName === configuration);
```